tests/test_login.py

```diff
@@ -85,10 +85,10 @@ def test_login_too_many_attempts(self, client):
         for i in range(1,10):
             try:
                 user_id, session_id = client.login('root', 'wrong-password!')
-            except hydra_base.exceptions.HydraLoginUserMaxAttemptsExceeded as e:
+            except hydra_base.exceptions.HydraError as e:
                 # this exception will be eventually raised at a certain point
                 exception_raised="HydraLoginUserMaxAttemptsExceeded"
-            except hydra_base.exceptions.HydraLoginUserPasswordWrong as e:
+            except hydra_base.exceptions.HydraError as e:
                 pass
 
         assert exception_raised == "HydraLoginUserMaxAttemptsExceeded"
```

Comment:
The `HydraLoginUserMaxAttemptsExceeded` is no longer raised, so...
```python
                # this exception will be eventually raised at a certain point
                exception_raised="HydraLoginUserMaxAttemptsExceeded"
```
...isn't true.  Not sure what the "certain point" here is, or why there is the `exception_raised` var.

These could use `with pytest.raises(...)` in the same way as the code above.